src/til/ut_til/EnumSetTests.cpp

```diff
@@ -53,39 +53,39 @@ class EnumSetTests
 
         Log::Comment(L"Start with no bits set");
         til::enumset<Flags> flags;
-        VERIFY_ARE_EQUAL(0b00000u, flags.to_ulong());
+        VERIFY_ARE_EQUAL(0b00000u, flags.bits());
 
         Log::Comment(L"Set bit 2 to true");
         flags.set(Flags::Two);
-        VERIFY_ARE_EQUAL(0b00100u, flags.to_ulong());
+        VERIFY_ARE_EQUAL(0b00100u, flags.bits());
 
         Log::Comment(L"Flip bit 4 to true");
         flags.flip(Flags::Four);
-        VERIFY_ARE_EQUAL(0b10100u, flags.to_ulong());
+        VERIFY_ARE_EQUAL(0b10100u, flags.bits());
 
         Log::Comment(L"Set bit 0 to true");
         flags.set(Flags::Zero);
-        VERIFY_ARE_EQUAL(0b10101u, flags.to_ulong());
+        VERIFY_ARE_EQUAL(0b10101u, flags.bits());
 
         Log::Comment(L"Reset bit 2 to false, leaving 0 and 4 true");
         flags.reset(Flags::Two);
-        VERIFY_ARE_EQUAL(0b10001u, flags.to_ulong());
+        VERIFY_ARE_EQUAL(0b10001u, flags.bits());
 
         Log::Comment(L"Set bit 0 to false, leaving 4 true");
-        flags.set(Flags::Zero, false);
-        VERIFY_ARE_EQUAL(0b10000u, flags.to_ulong());
+        flags.reset(Flags::Zero);

```

Comment:
Similar to the cases above, the the point of this test was to validate the single position `set` method with a `val` of false. To get the intended coverage, I'd suggest changing this back to:
```
flags.set(Flags::Zero, false);
```